sxs/waveforms/format_handlers/rotating_paired_diff_multishuffle_bzip2.py

```diff
@@ -61,6 +62,8 @@ def save(
     L2norm_fractional_tolerance : float, optional
         Tolerance passed to `WaveformModes.truncate`; see that
         function's docstring for details.  Default value is 1e-10.
+        If L2norm_fractional_tolerance is <=0.0 or None, then no
+        truncation is done.
```

Comment:
We should just skip the `None` possibility.  The old code that would have used that is gone anyway, so it won't break anything more.  If anything, letting it succeed but take a different path would be a bad thing.
```suggestion
        If L2norm_fractional_tolerance is <=0.0, then no
        truncation is done.
```